test/helpers/fixture_sets/hello_cloud.rb

```diff
@@ -21,6 +21,7 @@ def assert_all_up
       assert_stateful_set_up
       assert_job_up
       assert_network_policy_up
+      assert_secret_created
```

Comment:
why `_created` and not `_present`

Reply:
Addressed in https://github.com/Shopify/kubernetes-deploy/pull/438